### ISSUE TITLE
Fix tag cloud accessibility

### ISSUE THREAD

**Issue by jaguillette** [jaguillette/hvd-dh-omeka-theme#38]
The way that the actual inputs are hidden in the tag cloud makes them inaccessible to screen readers. They should be hidden in a better way that doesn't remove them from screen readers, and also that appropriately highlights focus on them.

**Comment by jaguillette** [comment]
finished per issue38 branch
